app/javascript/chat/message.jsx

```diff
@@ -32,22 +34,56 @@ const Message = ({
     />
   );
 
+  const ENTER_KEY_CODE = 13;
+  const dropdown = (
+    <div className="message__actions">
+      <span className="ellipsis__menubutton">
+        <img src={ThreeDotsIcon} alt="dropdown menu icon" />
+      </span>
+
+      <div className="messagebody__dropdownmenu">
+        <span
```

Comment:
Any reason this isn't an actual `<button />`? Using a `<button />` removes the need to have a `role` or `tabIndex` set and is more semantically correct. We also have a `<Button />` component in our [design system](https://storybook.dev.to/?path=/story/3-components-buttons--default) that is available to you.

You can import it like so:

```js
import { Button } from '@crayons'
```

@ludwiczakpawel, I don't think you've worked too much or at all in the chat portion of the application, but which button size would you recommend here?

Reply:
im not sure if it has to be our design system _Button_.. but if yes, then I'd suggest following:

```
<Button variant="ghost" ...>Edit</Button>
<Button variant="ghost-danger" ...>Delete</Button>
```